### PR TITLE
Fix tests broken by Spring Boot 2.4

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -128,7 +128,9 @@ commands:
               flyway:
                 target: latest
               redis:
-                url: redis://127.0.0.1:6379
+                host: 127.0.01
```
Comment (Nana-EC) [inline .circleci/config.yml:131]:
```suggestion
                host: 127.0.0.1
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (steven-sheehy) [reply .circleci/config.yml]: heh, not sure how that even worked. Fixed

File: .circleci/config.yml [hunk 2]
```diff
@@ -128,7 +128,9 @@ commands:
               flyway:
                 target: latest
               redis:
-                url: redis://127.0.0.1:6379
+                host: 127.0.0.1
+                password: ""
```
Comment (Nana-EC) [inline .circleci/config.yml:132]: nit: does leaving the password out set it as empty by default? If so we could do that but I'm guessing that results in an issue or an undesired default.
If this is the same as the other Redis config you should add that comment here to

**Review by Nana-EC — APPROVED** [review]
LGTM
One nit if applicable

**Review by ijungmann — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (steven-sheehy) [reply .circleci/config.yml:132]: I'm not sure, I didn't try every permutation of properties. There's some defaults in our config so I assumed it needed to be cleared and it worked. I'm trying to reduce the hardcoded config in CircleCI. I think the other comment in bootstrap.yml is sufficient.

**Pull request merged**